### PR TITLE
Update activations.py

### PR DESCRIPTION
**Pull request by shashank1623**
This file contains collection of activation function for use of machine learning models. These are function are based on tensorFlow library. The Code Contains activation function like:
1. gelu 
2. leaky_relu
3. relu
4. sigmoid
5. softmax
6. softplus
7. log_softmax
8. mish

as my knowledge tensorflow comes with some more activation function apart from these that are:
1. tanh(hyperbolic tangent)
2. selu (Scaled Exponential Linear Unit)
3. elu (Exponential Linear Unit)
4. hard_sigmoid

i have added all above stated activation function to activation.py file.

### CODE REVIEW

**Comment by shashank1623** [comment]
Hey can anyone guide me what i have done wrong here in the code.

**Comment by shashank1623** [comment]
Need some guidance why its failing. and how can i improve it.

**Comment by Daniel4078** [comment]
please first check out our contribution guide(https://lets-unify.ai/ivy/contributing/4_open_tasks.html#open-tasks) and then this TODO list issues(https://github.com/unifyai/ivy/issues/1569).

**Comment by Daniel4078** [comment]
Thank you for your efforts, but I have to say that you are not approaching the problem in the correct way. We have already setted up plans for adding all those functions and we have a complete guideline on how people like you can contribute in it. That pretty much covers the reason why the CI checks do not pass.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by shashank1623** [comment]
Currently working on this pr .

**Comment by Daniel4078** [comment]
@shashank1623 please, read my comments before and do not waste any more effort on this pull request as the format is not acceptable at all.
